pyproject.toml

```diff
@@ -10,7 +10,7 @@ build-backend = "setuptools.build_meta"
 write_to = "src/_pytest/_version.py"
 
 [tool.pytest.ini_options]
-minversion = "2.0"
+#minversion = "2.0"
```

Comment:
i think we want to bump this one a bit on a separate pr ;P

Reply:
Hahaha, I'm sorry, some hacks for local testing :facepalm: